app/controllers/pages.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+get '/' do
+  redirect '/dashboard' if session[:email]
```

Comment:
maybe have a method called `logged_in?` so that this line will read `redirect '/dashboard' if logged_in?`